### DISCUSSION TITLE
Are overlapping bounds allowed?

### DISCUSSION THREAD

**Discussion by taylor13** [cf-convention/discuss#360] [Q&A about using CF] [answered]
### Question

Suppose I want to save in a single variable, a vertically integrated quantity (e.g., heat content) extending from the surface of the ocean to various depths.  Is it legal to use the depth as the vertical coordinate and give it bounds [0, depth], so that it covers the region of interest?  (This would mean that each "grid cell" would contain all of the grid cells above it.)

I know there can be spaces between one grid cell and the next, or they can be contiguous (and share a bound), but can they overlap?

**Comment by JonathanGregory** [comment, marked as answer]
Yes, I believe they are allowed to overlap, since it's certainly not prohibited. A time axis for a monthly climatology made from a range of years would generally have overlapping bounds (although they're not ordinary bounds).

**Reply by davidhassell** [reply]
Just to add another example, in case it's of interest: A timeseries for consecutive days, where each value is a 5-day average centred on that day.
